iothub/device/tests/Authentication/IotHubConnectionStringBuilderTests.cs

```diff
@@ -133,28 +133,6 @@ public void IotHubConnectionStringBuilder_ParamConnectionString_ParsesSharedAcce
             csBuilder.UsingX509Cert.Should().BeFalse("SharedAccessSignature and X509 are mutually exclusive");
         }
 
-        [TestMethod]
-        public void IotHubConnectionStringBuilder_OverrideAuthMethodToken()
```

Comment:
The IAuthenticationMethod property is now readonly and can only be set in the constructor. Hence, this test is no longer required.

Same for the test below.